src/agent/src/Toolbox/Event/ToolCallSuccess.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Agent\Toolbox\Event;
+
+use Symfony\AI\Platform\Tool\Tool;
+
+/**
+ * Dispatched after successfully invoking a tool.
+ */
+final readonly class ToolCallSuccess
```

Comment:
either `ToolCallSuccessful` or `SuccessfulToolCall`? Same for the failed one.

WDYT @chr-hertel ?